src/Mod/Draft/importDXF.py

```diff
@@ -3860,7 +3860,7 @@ def export(objectslist, filename, nospline=False, lwPoly=False):
                     filename = filename.encode("utf8")
             dxf.saveas(filename)
 
-        FCC.PrintMessage("successfully exported " + filename + "\n")
+        FCC.PrintMessage("successfully exported" + " " + filename + "\n")
```

Comment:
Better to add a translate() here too I guess

Reply:
@yorikvanhavre there are no other `translate()`'s in the `PrintMessage()` in the rest of this file so maybe that would be a separate PR?